### PR TITLE
Timedistributed layer and Image Sequence Tutorial

### PR DESCRIPTION
**Pull request by tcapelle**
This module is equivalent to [Keras TimeDistributed Layer](https://keras.io/api/layers/recurrent_layers/time_distributed/). This wrapper allows to apply a layer to every temporal slice of an input. By default it is assumed the time axis (`tdim`) is the 1st one (the one after the batch size). A typical usage would be to encode a sequence of images using an image encoder.

This PR also adds a nice tutorial on adapting fastai to work on image squences, it is inspired from the siamese tutorial.
- UCR101 dataset setup
- Simple model baseline
- Image/LSTM model
- Using a Transformer based arch

### CODE REVIEW

**Comment by tcapelle** [comment]
I am currently making a tutorial notebook.

**Comment by jph00** [comment]
@tcapelle I'll wait until you've done the tutorial notebook so I can see which bits of that we might want to merge into this nb.

**Comment by tcapelle** [comment]
what do you think about moving all tutorials to one folder? So users can submit more tutorials.

**Comment by muellerzr** [comment]
I believe that's what examples/ is for

**Comment by tcapelle** [comment]
My english is not excellent but it is a first version. It may be better to put this notebooks elsewhere (examples??). Maybe move all tutorials there also?

**Comment by hamelsmu** [comment]
@tcapelle is this PR ready to review?  I see that it is still marked as draft

**Comment by tcapelle** [comment]
Not yet, let me add one example of the new transformer archs that have been recently invented (TimesFormer, ViVit, etc)

**Comment by jph00** [comment]
Sorry it took a while to merge this @tcapelle - it looks really great! :D

**Comment by tcapelle** [comment]
Thanks Jeremy, It may be a good idea to move the notebooks out of nbs to not make it run on the tests.

**Comment by muellerzr** [comment]
@tcapelle tutorials with all_slow flag won't run on github pushes (as it only runs `nbdev_test_nbs`. I'd still say we want them running with the --slow flag, as this way we know if a tutorial broke :)
